services/QuillLMS/client/app/bundles/Comprehension/components/studentView/promptStep.tsx

```diff
@@ -36,9 +36,11 @@ export default class PromptStep extends React.Component<PromptStepProps, PromptS
   constructor(props: PromptStepProps) {
     super(props)
 
+    const { submittedResponses, } = this.props
+
     this.state = {
-      html: this.formattedPrompt(),
-      numberOfSubmissions: 0,
+      html: this.formattedPrompt(submittedResponses),
+      numberOfSubmissions: submittedResponses.length || 0,
```

Comment:
if `submittedResponses` is an empty array, `submittedResponses.length` is going to evaluate to 0 anyway, and if it's null, calling `.length` will error, right? So I don't think the `|| 0` here is ever going to come into effect.